backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/AddDiskCommand.java

```diff
@@ -846,10 +846,18 @@ private void plugDiskToVmIfNeeded() {
     }
 
     protected boolean setAndValidateDiskProfiles() {
+        if (shouldSkipDiskProfilesValidation()) {
+            return true;
+        }
         return validate(diskProfileHelper.setAndValidateDiskProfiles(Collections.singletonMap(getDiskImageInfo(),
                 getStorageDomainId()), getCurrentUser()));
     }
 
+    private boolean shouldSkipDiskProfilesValidation() {
+        return getParameters().getParentCommand() == ActionType.CreateSnapshotForVm
+                || getParameters().getParentCommand() == ActionType.HibernateVm;
```

Comment:
would be better to check the type of the content of the disk - whether it's memory dump or memory metadata